src/components/Heading/Heading.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './Heading.scss';
+
+const CN = 'heading';
+
+const Heading = ({ title, position }) => (
+  <div className={`${CN}__wrapper`}>
+    <div className={`${CN}__line`} />
+    <div className={`${CN}__title ${position === 'right' ? 'right' : 'center'}`}>
```

Comment:
you can create some enum
const posEnum = {right: 'right', center: 'center'}

then use it just like that
```
    <div className={`${CN}__title ${posEnum[position]}`}>
```